Client/src/Pages/Settings/index.jsx

```diff
@@ -257,7 +257,14 @@ const Settings = ({ isAdmin }) => {
 									onClick={() =>
 										setIsOpen({ ...deleteStatsMonitorsInitState, deleteStats: true })
 									}
-									sx={{ mt: theme.spacing(4) }}
+									sx={{
+										mt: theme.spacing(4),
+										'&:hover': {
+										  backgroundColor: theme.palette.mode === 'light' 
+											? theme.palette.error.main 
+											: theme.palette.error.dark,
```

Comment:
These colors don't match the desing spec in #1335, they should aslo apply to all buttons, not just this specific one